GovUk.Frontend.AspNetCore.Extensions.ConformanceTests/TextOrHtmlHelper.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.AspNetCore.Html;
+using System.Text.Encodings.Web;
+
+namespace GovUk.Frontend.AspNetCore.Extensions.ConformanceTests
+{
+    public static class TextOrHtmlHelper
+    {
+        public static IHtmlContent? GetHtmlContent(string? text, string? html)
+        {
+            if (html != null)
```

Comment:
IMO it's better is use `is not` for null checks, as the `!=` operator can be overiden where `is not` can't be. Same argument for using the `string` alias instead of `String`

Reply:
And `is not` is used in other parts of the solution

Reply:
Agreed and updated.